metals/src/main/scala/scala/meta/internal/metals/Compilers.scala

```diff
@@ -172,8 +172,10 @@ class Compilers(
   }
 
   def load(paths: Seq[AbsolutePath]): Future[Unit] =
-    if (Testing.isEnabled) Future.successful(())
-    else {
+    if (Testing.isEnabled) {
+      scribe.info(s"LOAD $paths")
```

Comment:
Is this log needed? Could we maybe adjust the message to show what we are printing?

Reply:
oops, just leftovers from investigations. removed